node/service/src/lib.rs

```diff
@@ -866,7 +932,13 @@ where
 				telemetry.clone(),
 			);
 			proposer_factory.set_soft_deadline(SOFT_DEADLINE_PERCENT);
-			proposer_factory.enable_ensure_proof_size_limit_after_each_extrinsic();
+			// TODO: Need to cherry-pick 
+			//
+			// https://github.com/moonbeam-foundation/substrate/commit/
+			// d59476b362e38071d44d32c98c32fb35fd280930#diff-a1c022c97c7f9200cab161864c06d204f0c
+			// 8b689955e42177731e232115e9a6f
+			//
+			// proposer_factory.enable_ensure_proof_size_limit_after_each_extrinsic();
```

Comment:
Reminder to check this case and cherry-pick the necessary

Reply:
Unbroken link:

https://github.com/moonbeam-foundation/substrate/commit/d59476b362e38071d44d32c98c32fb35fd280930#diff-a1c022c97c7f9200cab161864c06d204f0c8b689955e42177731e232115e9a6f

Reply:
Is this still relevant? You might want to add it to the `TODO` section in the PR desc if so